tests/test_split_integer.py

```diff
@@ -1,21 +1,52 @@
 from app.split_integer import split_integer
+from random import randint
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
```

Comment:
This test is very difficult to understand
It's better to just write assertions with concrete values
In this case, it might be:
```
assert sum(split_integer(13, 5)) == 13
```
Do not write tests to assertions that check the same thing